doajtest/xml_upload_test_package/DOAJ/withdrawn_journal.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<records>
+
+    <record>
+        <language>fre</language>
+        <publisher>Codicille éditeur et CRILCQ</publisher>
+
+        <journalTitle>Successful</journalTitle>
```

Comment:
Maybe we should change to the title of this to "withdrawn" or something so it's obvious